frontend/src/components/header.tsx

```
@@ -0,0 +1,39 @@
+
+export const Header: React.FC<HeaderProps> = ({ dbname }) => {
+  const dispatch = useDispatch();
+
+  const reset_to_main = () => {
+    dispatch(update_table({ id: "" }));
+  };
+
+  return (
+    <nav className="header">
+      <section className="image_title_container">
+        <img src={dbimg} className="dbimg_header" onClick={reset_to_main} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)